src/updatechecker.cpp

```diff
@@ -219,6 +221,39 @@ UpdateChecker::UpdateChecker(): Thread("WinSparkle updates check")
 {
 }
 
+bool UpdateChecker::CreateInstallerProcess(const std::wstring filePath, DWORD dwCreationFlags, LPPROCESS_INFORMATION lpProcessInformation)
+{
+    STARTUPINFO             Startup;
+    PROCESS_INFORMATION     processInfo;
+
+    if (filePath == L"")
+    {
+        return false;
+    }
+
+    GetStartupInfo(&Startup);
+    Startup.wShowWindow = SW_SHOWNORMAL;
+    if (lpProcessInformation == NULL)
```

Comment:
Maybe a comment here would be nice about why a local Process Info is desirable to use when there is no caller provided Process Info.